### DISCUSSION TITLE
Novice help please!

### DISCUSSION THREAD

**Discussion by jackroutledge** [jef/streetmerchant#1037] [Help]
Ive installed as per the intructions (best i can) i dont have any .env file though. Please can someone help. Trying to source a PS5 in the UK

**Comment by Duck3y** [comment]
You should have a .env_example file. Copy this, edit it to your liking and save as .env

**Comment by jackroutledge** [comment]
@Duck3y its not in there, i deleted it and tried again with no luck. Im on mac if it makes a difference?

**Comment by Duck3y** [comment]
@jackroutledge I believe I saw another user mention that on Mac you will need to enable hidden files/folders.

https://appleinsider.com/articles/18/07/27/how-to-see-hidden-files-and-folders-in-macos

**Comment by jackroutledge** [comment]
@Duck3y  Brilliant, thankyou so much. Got that but still cant work out how to get PS5 notifications to my discord if you know how by any chance? Im in UK 

Thanks again

**Comment by Duck3y** [comment]
Are you making sure to use your webhook URL and the ID for the group you want to notify?

**Comment by jackroutledge** [comment]
Yeah I did get the webhook URL but not sure how I setup the app to check
for PS5 Uk stock. Just seem to get errors :(

**Comment by jackroutledge** [comment]
I have downloaded it on my windows computer now. Got it all installed but not sure how to set up the .env file

**Comment by Duck3y** [comment]
Post the env file you have made, make sure to leave out your discord webhook URL or any other private info.

**Comment by huntinganswers** [comment]
its good you have found some people to maybe help, but this is real basic stuff.  I suggest you read through the instructions as they are quite easy to decipher.  If there is a line you don't need - delete it.

install the streetmerchant script on your desktop so its easy to access.

I suggest you make a copy of the env-example file and stick it on your desktop.  Create another env file with notepad by simply saving it as .env.  You have to type .env.

when you're satisfied with the strings on your desktop env-example file.. copy the lines and paste to your .env file that you created with notepad earlier which is located in the root streetmerchant folder.

btw - you could also thank the people attempting to help.

**Comment by jackroutledge** [comment]
@Duck3y thankyou I think I am doing something completely wrong. Like I say im trying to get it working for in the UK for a PS5 disk. Ive taken out my webhook url

@huntinganswers im trying to learn.. its basic when you know how!

**Comment by jef** [comment]
Please keep it civil. Thank you :)

**Comment by jackroutledge** [comment]
# ** All configuration variables are optional **
# Read https://github.com/jef/streetmerchant#customization for help on customizing this file
#############################################################################################

ASCII_BANNER=""
ASCII_COLOR=""
AUTO_ADD_TO_CART=""
BROWSER_TRUSTED=""
COUNTRY="UK"
DESKTOP_NOTIFICATIONS=""
DISCORD_NOTIFY_GROUP=""
DISCORD_WEB_HOOK=""
EMAIL_PASSWORD=""
EMAIL_TO=""
EMAIL_USERNAME=""
HEADLESS=""
IN_STOCK_WAIT_TIME=""
LOG_LEVEL=""
LOW_BANDWIDTH=""
MAX_PRICE_SERIES_3060TI=""
MAX_PRICE_SERIES_3070=""
MAX_PRICE_SERIES_3080=""
MAX_PRICE_SERIES_3090=""
MAX_PRICE_SERIES_CORSAIR_SF=""
MAX_PRICE_SERIES_RX6800=""
MAX_PRICE_SERIES_RX6800XT=""
MAX_PRICE_SERIES_RX6900XT=""
MAX_PRICE_SERIES_RYZEN5600=""
MAX_PRICE_SERIES_RYZEN5800=""
MAX_PRICE_SERIES_RYZEN5900=""
MAX_PRICE_SERIES_RYZEN5950=""
MAX_PRICE_SERIES_SONYPS5C=""
MAX_PRICE_SERIES_SONYPS5DE=""
MAX_PRICE_SERIES_TEST=""
MICROCENTER_LOCATION=""
MQTT_BROKER_ADDRESS=""
MQTT_BROKER_PORT=""
MQTT_CLIENT_ID=""
MQTT_PASSWORD=""
MQTT_QOS=""
MQTT_TOPIC=""
MQTT_USERNAME=""
NVIDIA_ADD_TO_CART_ATTEMPTS=""
NVIDIA_SESSION_TTL=""
OPEN_BROWSER=""
PAGE_BACKOFF_MIN=""
PAGE_BACKOFF_MAX=""
PAGE_SLEEP_MIN=""
PAGE_SLEEP_MAX=""
PAGE_TIMEOUT=""
PAGERDUTY_INTEGRATION_KEY=""
PAGERDUTY_SEVERITY=""
PHILIPS_HUE_API_KEY=""
PHILIPS_HUE_CLOUD_ACCESS_TOKEN=""
PHILIPS_HUE_CLOUD_CLIENT_ID=""
PHILIPS_HUE_CLOUD_CLIENT_SECRET=""
PHILIPS_HUE_CLOUD_REFRESH_TOKEN=""
PHILIPS_HUE_LAN_BRIDGE_IP=""
PHILIPS_HUE_LIGHT_COLOR=""
PHILIPS_HUE_LIGHT_IDS=""
PHILIPS_HUE_LIGHT_PATTERN=""
PHONE_CARRIER=""
PHONE_NUMBER=""
PLAY_SOUND=""
PROXY_ADDRESS=""
PROXY_PROTOCOL=""
PROXY_PORT=""
PUSHBULLET=""
PUSHOVER_TOKEN=""
PUSHOVER_USER=""
PUSHOVER_PRIORITY=""
SCREENSHOT=""
SHOW_ONLY_BRANDS="SONY"
SHOW_ONLY_MODELS="PLAYSTATION 5, PS5"
SHOW_ONLY_SERIES="STANDARD,DISC"
SLACK_CHANNEL=""
SLACK_TOKEN=""
SMTP_ADDRESS=""
SMTP_PORT=""
STORES=""
TELEGRAM_ACCESS_TOKEN=""
TELEGRAM_CHAT_ID=""
TWILIO_ACCOUNT_SID=""
TWILIO_AUTH_TOKEN=""
TWILIO_FROM_NUMBER=""
TWILIO_TO_NUMBER=""
TWITCH_ACCESS_TOKEN=""
TWITCH_CHANNEL=""
TWITCH_CLIENT_ID=""
TWITCH_CLIENT_SECRET=""
TWITCH_REFRESH_TOKEN=""
TWITTER_ACCESS_TOKEN_KEY=""
TWITTER_ACCESS_TOKEN_SECRET=""
TWITTER_CONSUMER_KEY=""
TWITTER_CONSUMER_SECRET=""
TWITTER_TWEET_TAGS=""
USER_AGENT=""
WEB_PORT=""

**Comment by Duck3y** [comment]
SHOW_ONLY_BRANDS=""
SHOW_ONLY_MODELS="ps5 console, ps5 digital"
SHOW_ONLY_SERIES="sonyps5c, sonyps5de"

Remove sonyps5de and ps5 digital if you don't want to look for the digital version
